.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 120,
```

Comment:
Can we please change the setting here as well (see my comment below on ``proseWrap``)?

Reply:
(ah, you are probably in-the-works, seems you are making very incremental pushes 😄)

Reply:
Soooo, should i make it `80` then? what about `100` at least 🤣